js/projects.js

```diff
@@ -0,0 +1,240 @@
+  id: 'work-1',
+  alt: 'Work One Preview',
+  title: 'Tonic',
+  client: 'CANOPY',
+  job: 'Back End Dev',
+  year: '2015',
+  img: 'img/works/work1.png',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  tag1: 'html',
+  tag2: 'css',
+  tag3: 'javaScript',
+  btnId: 'work-btn-1',
+  details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
+
+const project2 = {
+  id: 'work-2',
+  alt: 'Work Two Preview',
+  title: 'Multi-Post Stories',
+  client: 'CANOPY',
+  job: 'Back End Dev',
+  year: '2015',
+  img: 'img/works/work2.png',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  tag1: 'html',
+  tag2: 'css',
+  tag3: 'javaScript',
+  btnId: 'work-btn-2',
+  details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
+
+const project3 = {
+  id: 'work-3',
+  alt: 'Work Three Preview',
+  title: 'Tonic',
+  client: 'CANOPY',
+  job: 'Back End Dev',
+  year: '2015',
+  img: 'img/works/work3.png',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  tag1: 'html',
+  tag2: 'css',
+  tag3: 'javaScript',
+  btnId: 'work-btn-3',
+  details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
+
+const project4 = {
+  id: 'work-4',
+  alt: 'Work Four Preview',
+  title: 'Tonic',
+  client: 'CANOPY',
+  job: 'Back End Dev',
+  year: '2015',
+  img: 'img/works/work4.png',
+  description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  tag1: 'html',
+  tag2: 'css',
+  tag3: 'javaScript',
+  btnId: 'work-btn-4',
+  details: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+};
+
+const projects = [project1, project2, project3, project4];
```

Comment:
**Kindly work on these required changes:**
(Use objects to store and access data.)

## In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.**

- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.


_I can recommend you, just store your projects in an array and then iterate each element_.

#

## Provide the requested design by your Figma Template:

**_Yours:_**
![Screen Shot 2021-12-14 at 18 25 37](https://user-images.githubusercontent.com/22258274/146100650-a852de91-e587-461f-b759-338cd69862eb.png)

**_Figma Template:_**

![Screen Shot 2021-12-14 at 18 25 15](https://user-images.githubusercontent.com/22258274/146100691-099505d0-f30b-48a2-91a8-e475707c507e.png)

Reply:
Hello @carlos-ssh 

Could you explain please why you think I'm not meeting the requirement "(Use objects to store and access data)" since all the information was indeed inside objects?